### ISSUE TITLE
Price library is not working

### ISSUE THREAD

**Issue by christroutner** [Bitcoin-com/bitbox-sdk#89]
Trying to use the BITBOX.Price library returns a bunch of errors and HTML. 

Here is an example node.js application to reproduce this issue:
```javascript
const BITBOXSDK = require('bitbox-sdk')
const BITBOX = new BITBOXSDK()

async function myfunc() {
  try {
    let current = await BITBOX.Price.current();
    console.log(current);
  } catch(error) {
   console.error(error)
  }
}
myfunc()
```

This is based on the BITBOX documentation:
https://developer.bitcoin.com/bitbox/docs/price

**Comment by MaxHastings** [comment]
In addition to that bug here is another method that returns HTML. From this tutorial. https://developer.bitcoin.com/tutorials/bitbox-cli/

`BITBOX.Network.getNetworkInfo().then((result) => { console.log(result); }, (err) => { console.log(err); });`

**Comment by cgcardona** [comment]
@MaxHastings we were advised to remove that endpoint so it no longer exists. That's why we no longer have a `Network` class here. I'll update that tutorial to remove that example

**Comment by cgcardona** [comment]
Blockchair has removed their bulk price endpoint so we're removing that functionality. 

This is resolved in BITBOX SDK v5.1.3

https://developer.bitcoin.com/bitbox/docs/price
